documentation/examples/wildfly-quickstart/pom.xml

```diff
@@ -6,14 +6,13 @@
 
     <parent>
         <groupId>org.hibernate.ogm</groupId>
-        <artifactId>hibernate-ogm-featurepack</artifactId>
+        <artifactId>hibernate-ogm-documentation-examples</artifactId>
```

Comment:
A suggestion, try to be consistent when you choose a name.
If we have decided to call it wildfly quickstart let's use the same name consistently. This is also helpful for us to check where the name doesn't fit right. 

Right now you have used:
1. `Wildfly-quickstart` for the folder name
2. `documentation-examples` for the artifact id
3. `Demonstration module` for the artifact name